service_objects/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
```

Comment:
This shouldn't be here.  It's a test model but you've stuck it inside service_objects/ which is what gets packaged and distributed.  If you need a model for your testing, you should create it in tests/test_services.py